bapsflib/utils/__init__.py

```diff
@@ -11,10 +11,24 @@
 """
 Package of developer utilities.
 """
-__all__ = ['decorators', 'errors', 'warnings']
+__all__ = ['decorators', 'errors', 'warnings', 'NdarrayToXarray']
 
 from . import (decorators, errors, warnings)
 
 
-def foo(x):
-    return x
+def NdarrayToXarray(data, arr_name={}):
```

Comment:
```suggestion
def ndarray_to_xarray(data, arr_name={}):
```

`bapsflib` follows PEP8 standards where [function names should be lowercase with underscores](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)

**Note:**  You'll also need to update the name in `__all__`.